### DISCUSSION TITLE
Update themes without having to restart apps

### DISCUSSION THREAD

**Discussion by jm355** [tsujan/Kvantum#909] [Q&A]
When I set a light or dark theme with qt5ct it updates on the fly, but kvantum doesn't do that. Is there any way to initiate a theme refresh without restarting apps?

**Comment by tsujan** [comment]
No. Kvantum can't (and shouldn't) refresh apps; it's just a *style* plugin (usually installed under `/usr/lib/qt/plugins/styles/`).

It's the "Platform Themes" plugin that can do that; it's usually installed under `/usr/lib/qt/plugins/platformthemes/` — although it might not be able to do it consistently when translucency changes, and that's why app restating , or logging out and in, is the best way.
